src/lib/card/card.scss

```diff
@@ -28,7 +28,7 @@
 // Raised
 //
 
-:host([raised]) {
+:host(:where([raised], :state(raised), :--raised)) {
```

Comment:
I'm having second thoughts about using both selectors. Chrome supports the final spec (`:state()`) as of last May and Safari and Firefox never used the old syntax.